src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -29,7 +29,7 @@
     protected String courseId;
     protected String feedbackSessionName;
     protected FeedbackSubmissionEditPageData data;
-    protected String hasResponses;
+    protected boolean isHasResponse;
```

Comment:
This variable is never initialized? It should also be declared in the loop for each question, and not as an attribute of the class.